kafka-streams-serdes/src/main/java/org/hypertrace/core/kafkastreams/framework/serdes/proto/ProtoDeserializer.java

```diff
@@ -0,0 +1,24 @@
+package org.hypertrace.core.kafkastreams.framework.serdes.proto;
+
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.protobuf.Message;
+import com.google.protobuf.Parser;
+import org.apache.kafka.common.serialization.Deserializer;
+
+public class ProtoDeserializer<T extends Message> implements Deserializer<T> {
```

Comment:
Can you please add a documentation comment explaining why this is required and how is it different from the (Kafka) built-in one?